src/tests/python_tests.yaml

```diff
@@ -0,0 +1,845 @@
+        # resizing mmap doesn't work (raises [Errno 48] Out of memory)
+      - test_basic
+      - test_offset
+      - test_resize_past_pos
+- test_module
+- test_modulefinder
+- test_msilib
+- test_multibytecodec
+- test_multiprocessing_fork
+- test_multiprocessing_forkserver
+- test_multiprocessing_main_handling
+- test_multiprocessing_spawn
+- test_named_expressions
+- test_netrc
+- test_nis
+- test_nntplib
+- test_ntpath:
+    # same set of failures as in test_posixpath
+    skip:
+      - test_exists_fd
+      - test_samestat_on_link
+      - test_samefile_on_link
+      - test_sameopenfile
+      - test_nonascii_abspath
+- test_numeric_tower
+- test_opcache
+- test_opcodes
+- test_openpty:
+    xfail: platform-specific
+- test_operator
+- test_optparse
+- test_ordered_dict
+- test_os:
+    xfail: |
+      segfault-eventfd; if we screen out the eventfd tests, we see 52 errors,
+      22 failures out of 312 tests
+- test_ossaudiodev
+- test_osx_env
+- test_pathlib:
+    skip:
+      - test_hardlink_to
+      - test_home
+      - test_link_to
+      - test_is_socket_true
+      - test_readlink
+      - test_open_mode
+      - test_touch_mode
+- test_patma
+- test_pdb:
+    xfail: 21/51 use subprocess
+- test_peepholer
+- test_peg_generator.test_c_parser
+- test_peg_generator.test_first_sets
+- test_peg_generator.test_grammar_validator
+- test_peg_generator.test_pegen
+- test_pickle:
+    xfail: dbm
+- test_picklebuffer
+- test_pickletools:
+    xfail: dbm
+- test_pipes
+- test_pkg
+- test_pkgutil
+- test_platform:
+    skip:
+      - test_architecture_via_symlink # fork
+- test_plistlib
+- test_poll:
+    xfail: subprocess
+- test_popen:
+    xfail: subprocess
+- test_poplib:
+    xfail: sockets
+- test_positional_only_arg
+- test_posix:
+    xfail: missing pwd module
+- test_posixpath:
+    skip:
+      - test_exists_fd
+      - test_samestat_on_link
+      - test_samefile_on_link
+      - test_nonascii_abspath
+- test_pow
+- test_pprint
+- test_print
+- test_profile
+- test_property
+- test_pstats
+- test_pty
+- test_pulldom
+- test_pwd
+- test_py_compile:
+    skip:
+      - test_stdin # fork
+- test_pyclbr
+- test_pydoc:
+    skip:
+      - test_server # fork
+      - test_synopsis_sourceless # expects __pycache__
+- test_pyexpat
+- test_queue:
+    # More than half of these make a thread
+    xfail: threads
+- test_quopri:
+    skip:
+      - test_scriptdecode # fork
+      - test_scriptencode # fork
+- test_raise
+- test_random:
+    skip:
+      - test_after_fork
+- test_range
+- test_re:
+    skip:
+      - "*locale*"
+- test_readline
+- test_regrtest:
+    xfail: All tests fork
+- test_repl:
+    xfail: All tests fork
+- test_reprlib
+- test_resource
+- test_richcmp
+- test_rlcompleter
+- test_robotparser:
+    xfail: socket
+- test_runpy:
+    skip:
+      - test_pymain_run* # fork
+- test_sax
+- test_sched:
+    skip:
+      - "*concurrent" # threads
```

Comment:
Could you point me to where the wildcard is managed for skip (couldn't find it)?  Thanks!